_pages/yaml-file-format.md

```diff
@@ -0,0 +1,176 @@
+The top-level mapping should have a blank line between each mapping entry.
+
+To ease machine identification of YAML blocks, [document markers](https://yaml.org/spec/1.2.2/#912-document-markers) are recommended. In particular, documents should begin with a YAML directive "`%YAML 1.2`" and directives end marker "`---`" and end with a document end marker "`...`".
+
+## YAML document schemata
+
+Each YAML document is a `map` with `str` keys.
+Each has a key "`type`" with one of a small set of strings, listed below, as its value.
+For readability, the `type` should be the first entry in the YAML document and both the key and value should be provided as plain scalars.
+
+The valid `type`s are
+
+- `structure`
+- `enumeration`
+- `calendar`
+- `month`
+- `datatype`
+- `uri`
+
+| Key | Value Type | Required by `type` | Permitted by `type` |
+|-----|------------|-------------|--------------|
+| `calendars` | `seq` of *uri* | `month` | |
+| `descriptions` | `seq` of `str` | all | |
+| `documentation` | `seq` of *uri* | | all |
+| `enumeration values` | `map` of *tag* to *uri*  | | `structure` |
+| `explanation` | `map` of *lang* to `str` | | all |
+| `extension tags` | `seq` of *extTag* | | `structure`, `enumeration`, `calendar`, `month` |
+| `label` | `map` of *lang* to `str` | | all |
+| `months` | `seq` of *uri* | `calendar` | |
+| `payload` | *type* | `structure` |
+| `standard tag` | *stdTag* | *special* | *special* | 
+| `substructures` | `map` of *uri* to *card* | `structure` | |
+| `superstructures` | `map` of *uri*\* to *card* | `structure` | |
+| `type` | `str` from a set listed here | all | |
+| `uri` | *uri* | all | |
+| `used by` | `seq` of *uri* | `enumeration` | |
+
+\* `superstructures` (only) permits the string "`HEAD psuedostructure`" as if it were a URI
+
+In the above table,
+
+- *stdTag* means a `str` matching production `stdTag`
+- *extTag* means a `str` matching production `extTag`
+- *tag* means a `str` matching production `tag`
+- *uri* means a `str` that is a URI
+- *lang* means a `str` in the datatype Language
+- *type* means one of the following:
+    - `null`
+    - a `str` that is the URI of a datatype
+    - a `str` that is the URI of a structure type delimited by `@` characters
+    - the specific `str` "`Y|<NULL>'"
+- *card* means one of the following four `str`: "`{0:1}`", "`{0:M}`", "`{1:1}`", or "`{1:M}`"
+
+Notes about specific items follow:
+
+- `calendars` is a list (in no particular order) of the URIs of calendars that use this month.
+    
+    The list may be incomplete, as a new calendar might be defined that uses an existing month.
+
+- `descriptions` is a list (in no particular order) of descriptions of the concept the YAML document is defining. Descriptions may be given in any language; it is recommended that at least one English-language description be included.
```

Comment:
Where is the IANA language tag that identifies what language a description is in?